saltstack/base/spryker/files/etc/nginx/spryker/backendapi.conf

```diff
@@ -0,0 +1,39 @@
+###
+### Common backendapi settings (to be included in vhost definition)
+### This file is maintained by salt
+###
+
+    # Timeout for backendapi requests - 10 minutes
+    # (longer requests should be converted to jobs and executed via jenkins)
+    proxy_read_timeout 600s;
+    proxy_send_timeout 600s;
+    fastcgi_read_timeout 600s;
+    client_body_timeout 600s;
+    client_header_timeout 600s;
+    send_timeout 600s;
+
+    # Static files can be delivered directly
+    location ~ (/assets/|/favicon.ico|/robots.txt) {
+        access_log        off;
+        expires           30d;
+        add_header Pragma public;
+        add_header Cache-Control "public, must-revalidate, proxy-revalidate";
+        try_files $uri =404;
+    }
+
+    # PHP application gets all other requests
+    location / {
+        add_header X-Server $hostname;
+        fastcgi_pass backend-$application_env-backendapi;
+        fastcgi_index index.php;
+        include /etc/nginx/fastcgi_params;
+        fastcgi_param SCRIPT_NAME /index.php;
+        fastcgi_param APPLICATION_ENV $application_env;
+        fastcgi_param APPLICATION_STORE $application_store;
+        fastcgi_param SPRYKER_PRODUCT_CONFIGURATOR_HOST demo-date-time-configurator.spryker.local;
+        fastcgi_param SPRYKER_PRODUCT_CONFIGURATOR_PORT 80;
+        fastcgi_param SPRYKER_PRODUCT_CONFIGURATOR_ENCRYPTION_KEY change123;
+        fastcgi_param SPRYKER_PRODUCT_CONFIGURATOR_HEX_INITIALIZATION_VECTOR 0c1ffefeebdab4a3d839d0e52590c9a2;
```

Comment:
Do we need those here?

Reply:
I really don't know, was from previous config, do you require changes or want me to remove it?